modules/apps/oauth2-provider/oauth2-provider-web/package.json

```diff
@@ -1,6 +1,8 @@
 {
 	"dependencies": {
+		"pkce-challenge": "^3.0.0"
```

Comment:
Because you added this, I would expect the yarn.lock to also be updated. Can you try running `yarn add pkce-challenge` from this module's directory, I believe that should update the yarn.lock at `/modules`.

Reply:
fixed